### PR TITLE
Updated README file

### PR DESCRIPTION
**Pull request by shivramsrivastava**
While we are in the process of setting things up this README Not will refrain other from raising issues.

### CODE REVIEW

**Comment by ICGog** [comment]
Can we please not approve our own changes? This is bad practice, and circumvents the entire review process.

I personally think it's fine for people to raise issues because they may stumble upon build issues that we haven't yet encountered.

**Comment by dhilipkumars** [comment]
@shivramsrivastava yes i tend to agree with @ICGog please copy any of us incase you want to get a PR in.  The merge bot will not accept if the author approves his own PR.  

In addition to that, the project should be fully build now after godep and all the fixes that we recently added to it.  Did you try it with a fresh clone?

File: README.md [hunk 1]
```diff
@@ -4,7 +4,10 @@ Kubernetes.
 [![Build Status](https://travis-ci.org/camsas/poseidon.svg)](https://travis-ci.org/camsas/poseidon)
 [![Coverage Status](https://coveralls.io/repos/github/camsas/poseidon/badge.svg?branch=master)](https://coveralls.io/github/camsas/poseidon?branch=master)
 
-***Note: this repo contains an initial prototype, it may break at any time! :)***
+***Note:
```
Comment (dhilipkumars) [inline README.md]: @shivramsrivastava do you really want this PR? if not we can close this.  The repo should be buildable now.

**Comment by shivramsrivastava** [comment]
@dhilipkumars yes will close this PR as no longer required.
There is a separate PR which had updated README and other docs for development and installation of Poseidon on k8s.

**Pull request closed without merging**